assets/js/components/OverlayCard/index.stories.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * OverlayCard Component Stories.
+ *
+ * Site Kit by Google, Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import OverlayCard from '.';
+import ReaderRevenueManagerIntroductoryGraphicDesktop from '../../../svg/graphics/reader-revenue-manager-monetize-graphic-desktop.svg';
+import ReaderRevenueManagerIntroductoryGraphicMobile from '../../../svg/graphics/reader-revenue-manager-monetize-graphic-mobile.svg';
```

Comment:
Not an issue but we don't have to use the fully qualified names for imported images. `GraphicDesktop` and `GraphicMobile` is more than enough to understand the difference there.
```suggestion
import GraphicDesktop from '../../../svg/graphics/reader-revenue-manager-monetize-graphic-desktop.svg';
import GraphicMobile from '../../../svg/graphics/reader-revenue-manager-monetize-graphic-mobile.svg';
```